python/pyspark/ml/util.py

```diff
@@ -283,3 +333,201 @@ def numFeatures(self):
         Returns the number of features the model was trained on. If unknown, returns -1
         """
         return self._call_java("numFeatures")
+
+
+@inherit_doc
+class DefaultParamsWritable(MLWritable):
+    """
+    .. note:: DeveloperApi
+
+    Helper trait for making simple :py:class`Params` types writable.  If a :py:class`Params`
```

Comment:
Does this work with the missing colon?  Please make sure to generate the API docs and check them.  See https://github.com/apache/spark/tree/master/docs for instructions